api/dp-openapi.yaml

```diff
@@ -1002,6 +1133,50 @@ components:
       in: header
       name: X-API-KEY
 
+  parameters:
+    projectid:
+      name: projectid
+      in: path
+      description: ID of project
+      required: true
+      schema:
+        type: string
+
+    state_query:
+      name: state_query
+      in: query
+      description: Project state
+      required: true
+      schema:
+        type: string
+
+    pagename:
+      name: pagename
+      in: path
+      description: Name of page
+      required: true
+      schema:
+        type: string
+
+    pagestate_query:
+      name: pagestate_query
```

Comment:
Ditto this needs to match the code, so `pagestate`.

Reply:
also done.